engine/runtime/src/main/java/org/enso/interpreter/node/callable/InvokeMethodNode.java

```diff
@@ -145,50 +147,25 @@ Object doWarning(
       UnresolvedSymbol symbol,
       Object self,
       Object[] arguments,
-      @Cached MethodResolverNode methodResolverNode,
+      @CachedLibrary(limit = "10") TypesLibrary dispatch,
```

Comment:
Why is the `TypesLibrary` called `dispatch` here? I was wondering what is `dispatch.getType` becuase the name `dispatch` suggested to me this is some special `getType` used for dispatch, not the standard type library one.